src/simulation/environment/dentonFluxModel/dentonFluxModel.cpp

```diff
@@ -19,26 +19,16 @@
 
 #include "simulation/environment/dentonFluxModel/dentonFluxModel.h"
 #include "architecture/utilities/linearAlgebra.h"
-#include "architecture/utilities/astroConstants.h"
```

Comment:
I think removing this `*.h` is why the Windows build is not building this module.  Please add this include back in.

Reply:
Done.